src/Signup.js

```diff
@@ -200,7 +207,7 @@ class Register extends Signup {
         return stage.complete().then(function(request) {
             if (request.auth) {
                 console.log("Stage %s is returning an auth dict", stageName);
-                return self._tryRegister(request.auth);
+                return self._tryRegister(request.auth, request.poll_for_success);
```

Comment:
for example, we rely on the body of the 401 having certain fields. That certainly isn't specced anywhere.